app/styles/_accessibility.scss

```diff
@@ -64,7 +64,7 @@
     &:hover {
         color: $color-gray;
         background-color: $color-light;
-        border-color: $color-bg-gray;
+        border-color: $$color-gray-light;
```

Comment:
Looks like there's an extra `$` here. Also, is the text readable from an a11y perspective when hovering over the button? The background color is darker now.